components/calendar/WeeklyAvailabilityCalendar.tsx

```
@@ -0,0 +1,365 @@
+  const hourHeight = 60 * minuteHeight
+  const gridHeight = totalMinutes * minuteHeight
+  const hourMarks = Math.ceil(totalMinutes / 60)
+
+  const days = useMemo(
+    () => Array.from({ length: 5 }, (_, index) => addDays(weekStart, index)),
+    [weekStart]
+  )
+
+  const segmentsByDay = useMemo(() => {
+    const dayMap = new Map<number, EventSegment[]>()
+    days.forEach((_, index) => dayMap.set(index, []))
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)